### PR TITLE
Update _common.yaml

### PR DESCRIPTION
**Pull request by chuk-a**

### CODE REVIEW

File: sentences/mn/_common.yaml [hunk 1]
```diff
@@ -7,6 +7,53 @@ responses:
...
+    values:
+      - "цагаан"
+      - "хар"
+      - "улаан"
+      - "улбар шар"
+      - "шар"
+      - "ногоон"
+      - "цэнхэр"
+      - "ягаан"
+      - "бор"
```
Comment (schizza) [inline sentences/mn/_common.yaml:21]: there must be counterparts to stated values:
eg: 
```
lists:
  color:
     values:
        - in: "цагаан"  #color you are using in your language
        - out: "white"  #color name in English
```

File: sentences/mn/_common.yaml [hunk 2]
```diff
@@ -7,6 +7,53 @@ responses:
...
+  name: "[the] {name}"
+  area: "[the] {area}"
+  what_is: "(юу | хэд вэ | юу вэ)"
+  brightness: "{brightness} [percent]"
+  turn: "(асаа | нээ | өөрчил)"
+  temp: "(хэм | градус)"
+  temperature: "{temperature} [degrees] [{temperature_unit}]"
+  open: "(нээ | онгойлго)"
+  close: "(хаа | буулга | багасга)"
+  window: "(window[s] | blind[s] | curtain[s] | shutter[s])"
```
Comment (schizza) [inline sentences/mn/_common.yaml:56]: No spaces between `|` as new big changes will be added #830

**Review by schizza — CHANGES_REQUESTED** [review]
Stick to #830 rules

**Comment by frenck** [comment]
Since there was no response at all, I'm declaring this PR stale and closing it.
Feel free to re-open this PR when you feel like that is the appropriate thing to do.

../Frenck

**Pull request closed without merging**